We want that the zoom area follow the focus by default to help people
with a low vision, and we need to avoid using the same keyboard shortcut
in mag and opacify, or in mag and neg.

diff -ru compiz-plugins-main-master/metadata/mag.xml.in compiz-plugins-main-master.new/metadata/mag.xml.in
--- compiz-plugins-main-master/metadata/mag.xml.in	2021-10-30 17:36:42.000000000 +0200
+++ compiz-plugins-main-master.new/metadata/mag.xml.in	2022-03-08 18:23:38.454452176 +0100
@@ -18,7 +18,7 @@
 				<option name="initiate" type="key">
 					<short>Initiate</short>
 					<long>Zoom In</long>
-					<default>&lt;Super&gt;m</default>
+					<default>&lt;Super&gt;o</default>
 				</option>
 				<option name="zoom_in_button" type="button">
 					<short>Zoom In</short>
diff -ru compiz-plugins-main-master/metadata/opacify.xml.in compiz-plugins-main-master.new/metadata/opacify.xml.in
--- compiz-plugins-main-master/metadata/opacify.xml.in	2021-10-30 17:36:42.000000000 +0200
+++ compiz-plugins-main-master.new/metadata/opacify.xml.in	2022-03-08 18:25:24.918455222 +0100
@@ -15,7 +15,7 @@
         <option name="toggle_key" type="key">
           <short>Toggle Opacify</short>
           <long>Use this to enable/disable Opacify on the fly. Previously opacified windows will not be reset once you disable it like this.</long>
-          <default>&lt;Super&gt;o</default>
+          <default>&lt;Shift&gt;&lt;Super&gt;o</default>
         </option>
       </group>
       <group>
